### PR TITLE
Fix #948

### LINKED ISSUE #948: Manage action.id_root = ""
Following #942 and #943 , we need to find a way to change/enhance the way we check animation here :
https://github.com/KhronosGroup/glTF-Blender-IO/blob/19665b05a59550dee1aeaf24d548e2ce8477f4ba/addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animations.py#L51

Seems that id_root is always set to "OBJECT" or "KEY" when creating "by hand" in the UI, but sometimes the field is empty (for example when importing fbx file).

What I am proposing:
* Open a bug report for fbx importer
* in glTF exporter: Check the value of id_root. If set --> no change regarding the way it's handle for now in master. If id_root is empty, we need an extra search to be able to determinate if the action is an object/armature animation, or if this is a morph animation.

### PR DESCRIPTION
**Pull request by julienduroure**
Fix #948

### CODE REVIEW

**Comment by julienduroure** [comment]
@scurest Can you please have a look ?

File: addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animations.py [hunk 1]
```diff
@@ -53,10 +53,10 @@ def gather_animations(blender_object: bpy.types.Object,
                 break
 
     # Export all collected actions.
-    for blender_action, track_name in blender_actions:
+    for blender_action, track_name, on_type in blender_actions:
 
         # Set action as active, to be able to bake if needed
-        if blender_action.id_root == "OBJECT" and blender_object.animation_data: # Not for shapekeys!
+        if on_type == "OBJECT": # Not for shapekeys!
```
Comment (scurest) [inline addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animations.py]: Why isn't the `and blender_object.animation_data` check needed anymore?

**Comment by scurest** [comment]
This was originally to fix #835 IIRC? I'll retest it...

File: addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animations.py (hunk 1 above)
Comment (scurest) [reply addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animations.py]: Okay, I see `__get_blender_actions` can only return `on_type == 'OBJECT'` when it has already checked `blender_object.animation_data`.

**Pull request merged**